Sources/Sentry/Public/SentryOptions.h

```diff
@@ -456,7 +456,17 @@ NS_SWIFT_NAME(Options)
  * the SDK sets it to the default of @c 0.
  * This property is dependent on @c tracesSampleRate -- if @c tracesSampleRate is @c 0 (default),
  * no profiles will be collected no matter what this property is set to. This property is
- * used to undersample profiles *relative to* @c tracesSampleRate
+ * used to undersample profiles *relative to* @c tracesSampleRate .
+ * @note Setting this value to @c nil enables an experimental new profiling mode, called continuous
```

Comment:
`m`: I think we should mention that nothing happens if you don't interact with the continuous profiling API. Otherwise, you could assume that this is a breaking change.

Reply:
Otherwise, I think what you're saying is sort of implied by the explanation that's already there, which talks about needing to call SentrySDK.startProfiler